_includes/04-lists.md

```diff
@@ -0,0 +1,3 @@
+Emoji
+Ordered Lists
```

Comment:
Ordered Lists
Ordered lists have numbers. You can nest ordered lists within a list item by indenting them. You can read more about [formatting and syntax](https://help.github.com/articles/basic-writing-and-formatting-syntax/)[](https://github.com/githubteacher)[](https://github.com/marketplace/github-learning-lab)[](https://github.com/Rajeshbommu)[](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)[](https://github.com/githubteacher)[](https://github.com/) in the GitHub Help.

1. Item 1
2. Item 2
3. Item 3
   1. Item 3a
   2. Item 3b
Item 1
Item 2
Item 3
Item 3a
Item 3b